internal/pkg/levee/propagation/propagation.go

```diff
@@ -130,7 +130,7 @@ func (prop *Propagation) shouldNotVisit(n ssa.Node, maxInstrReached map[*ssa.Bas
 	return false
 }
 
-func (prop *Propagation) visit(n ssa.Node, maxInstrReached map[*ssa.BasicBlock]int, lastBlockVisited *ssa.BasicBlock) {
+func (prop *Propagation) taint(n ssa.Node, maxInstrReached map[*ssa.BasicBlock]int, lastBlockVisited *ssa.BasicBlock) {
```

Comment:
I think this should be called something like `taintNeighbors`.

Reply:
Done.